custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py

```diff
@@ -0,0 +1,496 @@
+                          if index.referenced_type == CASE_TYPE_OCCURRENCE]
+        occurrence_id = occurrence_ids[0] if occurrence_ids else None
+
+        latest_episode_id = max((case.opened_on, case.case_id) for case in episodes
+                                if is_episode_of_occurrence(case, occurrence_id)
+                                and not case.closed)[1]
+
+        test_type = episode.get_case_property('test_confirming_diagnosis')
+        props = {
+            'is_active': 'yes' if episode.case_id == latest_episode_id else 'no',
+            'dosage_display': episode.get_case_property('full_dosage'),
+            'dosage_summary': episode.get_case_property('full_dosage'),
+            'rft_general': 'diagnosis_dstb',
+            'diagnosis_test_type': test_type,
+            'diagnosis_test_type_label': TEST_TO_LABEL.get(test_type, ""),
+        }
+
+        treatment_status = episode.get_case_property('treatment_status')
+        if treatment_status == 'second_line_treatment':
+            props['treatment_status'] = 'initiated_second_line_treatment'
+        elif treatment_status == 'yes_phi':
+            props['treatment_status'] = 'initiated_first_line_treatment'
+        elif treatment_status == 'yes_private':
+            props['treatment_status'] = 'initiated_outside_facility'
+
+        if treatment_status == 'yes_phi':
+            props['treatment_initiated'] = 'yes_phi'
+        elif treatment_status == 'yes_private':
+            props['treatment_initiated'] = 'yes_private'
+        elif treatment_status:
+            props['treatment_initiated'] = 'no'
+
+        if not episode.get_case_property('date_of_diagnosis'):
+            props['date_of_diagnosis'] = episode.get_case_property('date_reported')
+
+        return CaseStructure(
+            case_id=episode.case_id,
+            walk_related=False,
+            attrs={
+                "create": False,
+                "update": props,
+            },
+        )
+
+    def migrate_test(self, test, person):
+        self.total_tests += 1
+        props = {
+            'is_direct_test_entry': 'no',
+            'rft_drtb_diagnosis': test.get_case_property('diagnostic_drtb_test_reason'),
+            'dataset': person.get_case_property('dataset'),
+        }
+
+        if test.get_case_property('follow_up_test_reason') == 'private_ntm':
+            props['rft_general'] = 'diagnosis_dstb'
+            props['rft_dstb_diagnosis'] = 'private_ntm'
+        else:
+            props['rft_general'] = {
+                'diagnostic': 'diagnosis_dstb',
+                'diagnosis_dstb': 'diagnosis_dstb',
+                'follow_up_dstb': 'follow_up_dstb',
+                'diagnosis_drtb': 'diagnosis_drtb',
+            }.get(test.get_case_property('purpose_of_testing'), "")
+            props['rft_dstb_diagnosis'] = test.get_case_property('diagnostic_test_reason')
+            props['rft_dstb_followup'] = test.get_case_property('follow_up_test_reason')
```

Comment:
@esoergel 

I added one more row (row 14) in the doc for episode_case_id to be set